gradle.properties

```diff
@@ -108,12 +108,16 @@ artifactRedirecting.androidx.compose.foundation.version=1.7.0-beta05
 artifactRedirecting.androidx.compose.material.version=1.7.0-beta05
 # Look for `COMPOSE_MATERIAL3_COMMON` in libraryversions.toml
 artifactRedirecting.androidx.compose.material3.common.version=1.0.0-alpha01
+# Look for `COMPOSE_MATERIAL3_ADAPTIVE` in libraryversions.toml
+artifactRedirecting.androidx.compose.material3.adaptive.version=1.0.0-alpha01
 # We use artifactRedirecting not only for Compose libs:
 artifactRedirecting.androidx.collection.version=1.4.0
 artifactRedirecting.androidx.annotation.version=1.8.0
 artifactRedirecting.androidx.lifecycle.version=2.8.0
 artifactRedirecting.androidx.navigation.version=2.8.0-beta05
 artifactRedirecting.androidx.savedstate.version=1.2.1
+# Look for `WINDOW` in libraryversions.toml
+artifactRedirecting.androidx.window.core.version=1.3.0-alpha01
```

Comment:
The stable 1.3.0 was release a few months ago. Any reason to use alpha?

Reply:
I didn't touch the version. As I understand the version was set in the commit we based our fork on

Reply:
window subfolder wasn't involved in specific version selection, If it's outdated, please merge the stable state (might be done separately)

Reply:
Actually, I'm almost sure that this version doesn't match to the commit/state of the code. Blocking the merge before explicit check